api.go

```diff
@@ -537,6 +543,26 @@ func (s *Stream) Nearest(ctx context.Context, time int64, version uint64, backwa
 	return
 }
 
+// Earliest returns the point nearest to the specified start time searching forward such that the
+// returned point will be >= after. This function is therefore essentially synonymous with the
+// Nearest function.
+func (s *Stream) Earliest(ctx context.Context, after int64, version uint64) (rv RawPoint, ver uint64, err error) {
+	if after == 0 {
+		after = MinimumTime
+	}
```

Comment:
So now this poses the same problem as before because it eliminates 0 as an actual value to start from to ignore negative time values. My preference would be to eliminate -1 instead of 0 (e.g. if -1 is passed then start at `MinimumTime`) unless you had other thoughts.

Reply:
Well actually I was going to suggest eliminating the after/before parameters and always passing Maximum/Minimum. Otherwise the utility of the Earliest/Latest functions is not that high (it saves one bool to Nearest)

Reply:
I'd still like the ability to limit earliest and latest to no future values and to omit historical data. And you're right, it's not saving anything on `Nearest` - Allen even called it "syntactic sugar", but it does make analytics code a lot more readable. If you're opposed to it, then we can just assume folks who are using the Go bindings are database users and not necessarily analysts and just keep this stuff in `btrdb-python` and the R and Julia bindings.

Reply:
Hmm. The python bindings don't take an after/before parameter either though. I'm in favor of Latest/Earliest in principle, I just think they should always apply to the whole stream (i.e. use Max/Min time). I just noticed the python bindings use 0 and Now() though, which I don't particularly like...